doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -363,10 +363,31 @@ Swift is the object storage management service, Swift depends on Keystone and po
 _The Swift backends include iSCSI drives, Ceph RBD and NFS._ 
 
 #### 4.3.1.5 Neutron
-Neutron is the networking service, Neutron depends on Keystone and has services running on the control nodes and the compute nodes:
--	neutron-api
--	neutron-rpc
--	neutron-*-agent agents which runs on Compute and Network nodes
+Neutron is the networking service, Neutron depends on Keystone and has services running on the control nodes and the compute nodes. Depending upon the workloads to be hosted by the Infrastructure and the expected load on the controller node, some of the Neutron services can run on separate network node(s). Factors affecting controller node load include number of compute nodes and the concurrent number of nova API calls as well as NAT. In this case, network nodes are widely added to manage L3 traffic for overlay tenant networks and interconnection with external networks. Table 4-2 below lists the networking service components and their placement. Please note that while network nodes are listed in the table below, network nodes only deal with tenant networks and not provider networks. Also, network nodes are not required when SDN is utilized for networking.
```

Comment:
``` Neutron depends on Keystone and has services``` -> Pls remove "depends on Keystone".

Reply:
@pgoyal01 do we need to explain Neutron arch ( which part be on compute and which one will be hosted on the controller ) https://docs.openstack.org/security-guide/networking/architecture.html 

can we remove this "Depending upon the workloads to be hosted by the Infrastructure and the expected load on the controller node"

Some of VNF requirement can be achieved by having some neutron services running on dedicated node called " Network node / Neutron server node " 

.....

Reply:
@msidana why? Keystone provide the identity service for Neutron as well and so Keystone should be present first and so Neutron depends on keystone

Reply:
I am not questioning the dependency of Neutron over keystone. Its just that this section is about Neutron so we should talk about neutron only instead of quoting the dependency here.

Reply:
@samuelhellec @msidana , as well, all OpenStack services depend on Keystone so no need to mention it explicitly here

Reply:
In term of coherence of chapter 4.3, each module start with this statement.
The information provided let you know in which order you have to rollout the different modules.

Reply:
In this PR no change has been made to the order of the services. This write-up is similar to what exists fr other services as @samuelhellec as indicated. 

The OpenStack services were introduced in Ch03, and the intent in Chapter 04 is to provide more detail including implementation aspects.  Hence the dependencies.

May I suggest that we approve this PR based on consistency of the materials. 

W.r.t. whether we should list dependencies, let us discuss in the RA-1 call, although this material was completed pre-Antwerp.

Reply:
Ok.